Library/Formula/chapel.rb

```diff
@@ -27,8 +27,9 @@ def install
     prefix.install_metafiles
 
     # Install chpl and other binaries (e.g. chpldoc) into bin/ as exec scripts.
-    bin.install Dir[libexec/"bin/darwin/*"]
-    bin.env_script_all_files libexec/"bin/darwin/", :CHPL_HOME => libexec
+    platform = OS::NAME << (OS.linux? ? Hardware::CPU.bits.to_s : "")
```

Comment:
I apologize that this line is a bit obtuse.

I also tried doing it in an interpolation style, but I couldn't seem to get it to work:
```ruby
platform = "#{OS::NAME}#{OS.linux? ? Hardware::CPU.bits.to_s : ''}"
```